### PR TITLE
Compatibility documentation initial revisions.

### PR DESCRIPTION
**Pull request by danieltprice**
Edits. Added links to extension documentation.

### CODE REVIEW

**Comment by stepashka** [comment]
looks good from my perspective!
@MMeent , @hlinnaka , could you review too?

**Review by stepashka — DISMISSED** [review]

File: content/docs/reference/compatibility.md [hunk 1]
```diff
@@ -72,14 +72,14 @@ select * from pg_settings where source <> 'default';
 | work_mem             |         |                                                                                           |
 | maintenance_work_mem |         |                                                                                           |
 
-## Unlogged tables
+## Unlogged Tables
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/reference/compatibility.md:75]: Hi Matthias, 
Good point. It's a mixed bag in this topic and in the docs as a whole. Let's go with sentence-style capitalization going forward.

File: content/docs/reference/compatibility.md [hunk 2]
```diff
@@ -4,65 +4,65 @@ redirectFrom:
   - /docs/conceptual-guides/compatibility
 ---
 
-Neon is protocol- and application-compatible with PostgreSQL. However, when you connect to our cloud offering, there are some limitations that you need to take into account.
+Neon is protocol and application-compatible with PostgreSQL. However, when using Neon, there are some limitations that you should be aware of.
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/reference/compatibility.md:7]: Ah, ok. I did not realize a distinction was be drawn here. I will revise.

[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/reference/compatibility.md]: Added content to address #146

**Comment by stepashka** [comment]
@MMeent , please review, as your concerns were addressed

**Review by stepashka — APPROVED** [review]

**Comment by danieltprice** [comment]
I missed the suggested revisions above before merging. I will commit those changes under a new branch.

File: content/docs/reference/compatibility.md [hunk 3]
```diff
@@ -4,82 +4,81 @@ redirectFrom:
...
+| [dict_int](https://www.postgresql.org/docs/14/dict-int.html)                |     1.0 |      |
+| [fuzzystrmatch](https://www.postgresql.org/docs/14/fuzzystrmatch.html)           |     1.1 |      |
+| [hstore](https://www.postgresql.org/docs/14/hstore.html)                  |     1.8 |      |
+| [intarray](https://www.postgresql.org/docs/14/intarray.html)                |     1.5 |      |
+| [isn](https://www.postgresql.org/docs/14/isn.html)                     |     1.2 |      |
+| [lo](https://www.postgresql.org/docs/10/lo.html)                      |     1.1 |      |
+| [ltree](https://www.postgresql.org/docs/14/ltree.html)                   |     1.2 |      |
+| [pg_trgm](https://www.postgresql.org/docs/14/pgtrgm.html)                 |     1.6 |      |
+| [pgcrypto](https://www.postgresql.org/docs/14/pgcrypto.html)                |     1.3 |      |
+| [plpgsql](https://www.postgresql.org/docs/current/plpgsql.html)                 |     1.0 | Pre-installed with Postgres |
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/reference/compatibility.md:40]: Thank you. I will make that change.

File: content/docs/reference/compatibility.md [hunk 4]
```diff
@@ -4,82 +4,81 @@ redirectFrom:
...
+| [postgis_raster](https://postgis.net/docs/raster.html)          |   3.3.0 |      |
+| [postgis_tiger_geocoder](http://postgis.net/docs/Geocode.html)  |   3.3.0 | Cannot be installed using the Neon web UI. Use your `psql` user credentials to install this extension instead. |
+| [postgis_topology](http://postgis.net/docs/manual-dev/Topology.html)        |   3.3.0 |      |
+| [seg](https://www.postgresql.org/docs/current/seg.html)                     |     1.4 |      |
+| [tablefunc](https://www.postgresql.org/docs/current/tablefunc.html)               |     1.0 |      |
+| [tcn](https://www.postgresql.org/docs/current/tcn.html)                     |     1.0 |      |
+| [tsm_system_rows](https://www.postgresql.org/docs/current/tsm-system-rows.html)         |     1.0 |      |
+| [tsm_system_time](https://www.postgresql.org/docs/current/tsm-system-time.html)         |     1.0 |      |
+| [unaccent](https://www.postgresql.org/docs/current/unaccent.html)                |     1.1 |      |
+| [uuid-ossp](https://www.postgresql.org/docs/current/uuid-ossp.html)               |     1.1 |      |
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/reference/compatibility.md:51]: Thank you. I will make that change.

File: content/docs/reference/compatibility.md [hunk 5]
```diff
@@ -4,82 +4,81 @@ redirectFrom:
...
 <a id="default-extensions/"></a>
 
-## Available PostgreSQL Extensions
+## Available Postgres extensions
 
-During the technical preview, Neon restricts the installation of PostgreSQL extensions.
+During the technical preview, Neon restricts the installation of Postgres extensions.
 
-The following PostgreSQL extensions can be installed by the user:
+Installation is permitted for the following Postgres extensions:
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/reference/compatibility.md:23]: Will go with PostgreSQL, which is the official name.

**Pull request merged**